systemtest/src/main/java/io/strimzi/systemtest/kafkaclients/externalClients/OauthExternalKafkaClient.java

```diff
@@ -81,6 +82,29 @@ protected Builder self() {
         }
     }
 
+    @Override
+    public AbstractKafkaClient.Builder<OauthExternalKafkaClient.Builder> toBuilder(OauthExternalKafkaClient oauthExternalKafkaClient) {
+        OauthExternalKafkaClient.Builder builder = new OauthExternalKafkaClient.Builder();
+
+        builder.withTopicName(oauthExternalKafkaClient.getTopicName());
+        builder.withPartition(oauthExternalKafkaClient.getPartition());
+        builder.withMessageCount(oauthExternalKafkaClient.getMessageCount());
+        builder.withNamespaceName(oauthExternalKafkaClient.getNamespaceName());
+        builder.withClusterName(oauthExternalKafkaClient.getClusterName());
+        builder.withConsumerGroupName(oauthExternalKafkaClient.getConsumerGroup());
+        builder.withKafkaUsername(oauthExternalKafkaClient.getKafkaUsername());
+        builder.withSecurityProtocol(oauthExternalKafkaClient.getSecurityProtocol());
+        builder.withCertificateAuthorityCertificateName(oauthExternalKafkaClient.getCaCertName());
+        builder.withProducerProperties(oauthExternalKafkaClient.getProducerProperties());
+        builder.withConsumerProperties(oauthExternalKafkaClient.getConsumerProperties());
+        builder.withOauthClientId(oauthExternalKafkaClient.getClientId());
+        builder.withClientSecretName(oauthExternalKafkaClient.getClientSecretName());
+        builder.withOauthTokenEndpointUri(oauthExternalKafkaClient.getOauthTokenEndpointUri());
+        builder.withIntrospectionEndpointUri(oauthExternalKafkaClient.getIntrospectionEndpointUri());
+
+        return builder;
+    }
+
```

Comment:
I guess we discussed this when I was working on the example clients builder but -> isn't there some way how to just set the "properties" that are specific for Oauth external clients here and the others set in the Abstract? I know that there can be some dependency (etc.) issues, but still asking...

Reply:
Thanks for catch, I have re-designed it a little bit and now in the client will not be a redundant code.

Reply:
Thanks a lot :)